src/views/homepage/components/pre-footer/pre-footer.module.css

```diff
@@ -0,0 +1,67 @@
+.preFooter {
+  --container-max-width: 1140px;
+  --grid-template-columns: 4;
+  --grid-column-gap: 24px;
```

Comment:
Still thinking about how to extract these vars for the homepage implementation. As I build out the rest of the page, I will have a better understanding of how to make these reusable. That will be handled in future PRs.